CHANGES.txt

```diff
@@ -11,6 +11,8 @@ Features
 * Sort posts in the following order (most important last): source path
   (A-Z), title (A-Z), date (reverse chronological order), priority
   meta number (descending). (Issue #2627)
+* Add META_GENERATOR_TAG option in conf.py allowing the meta generator 
+  to be disabled if needed. (Issue #2628)
```

Comment:
You need to add this more at the top, before the changelist for version v7.8.3 starts. Now it looks like this was added in v7.8.3, while it wasn't.

Just start an empty `Features` section at the top of the file and move the comment there.